app/javascript/packs/application.js

```diff
@@ -35,6 +35,7 @@ import SessionPrompt from "./components/SessionPrompt";
 import RefundApplicantPage from "./pages/refunds/ApplicantPage";
 import RefundFeesPage from "./pages/refunds/FeesPage";
 import ClaimDetailsPage from "./pages/claims/ClaimDetailsPage";
+import WritingYourClaimStatementPage from "./pages/claims/WritingYourClaimStatementPage";
```

Comment:
I thought we agreed that there was no such page as 'WritingYourClaimStatement' ?  If you look at the other javascript page modules, they all have the same name as the page where they are used.

We could invent like a 'component' module which we could use for partials, but it would need more thinking about as there may be more than one on a page, so ideally stick with page modules.

I can't remember what page this was used on else I would just say what to call it